flake.nix

```diff
@@ -149,10 +149,10 @@
             inputs.ogmios-datum-cache.defaultPackage.${system};
           ogmios = ogmios.packages.${system}."ogmios:exe:ogmios";
           ogmios-fixtures = ogmios;
-          purescriptProject = import ./nix { inherit system; pkgs = final; };
+          cardano-cli = cardano-node-exe.packages.${system}.cardano-cli;
```

Comment:
This input no longer exists -- did you forget to update the lockfile?

Reply:
Thanks and sorry about that. My understanding of the nix is fairly limited I was mostly just working from a commit which works and one which doesn't and pulling over the differences.